lib/main_production.dart

```diff
@@ -20,6 +21,7 @@ void main() async {
     await getIt<FirebaseService>().initialize();
   }
   await getIt<MatrixService>().init();
+  await getIt<AIClient>().init();
```

Comment:
don't need to call init here, let put logic of init function into AIClient constructor call